ui/src/shared/utils/formatStatValue.ts

```diff
@@ -26,15 +33,19 @@ export const formatStatValue = (
 
     const roundedValue = value.toFixed(digits)
 
-    localeFormattedValue = Number(roundedValue).toLocaleString(undefined, {
-      maximumFractionDigits: MAX_DECIMAL_PLACES,
-    })
+    localeFormattedValue =
+      Number(roundedValue) === 0
```

Comment:
what do you think of turning this code into a formatting function and including the negative zero check as part of it?

Reply:
This piece of code is actually inside a formatting function already. The ternary is to allow positive zero to have decimal places when user-specified.

Reply:
well then